obsah/__init__.py

```
@@ -359,6 +405,27 @@ def generate_ansible_args(inventory_path, args, obsah_arguments):
+        if persist_params:
+            for (reset_key, reset_values) in metadata['reset']:
+                if reset_key in persist_params and persist_params.get(reset_key) != getattr(args, reset_key):
+                    for arg in reset_values:
+                        if arg in persist_params and persist_params[arg] == getattr(args, arg):
+                            delattr(args, arg)
+            if args.obsah_reset:
+                for reset_arg in args.obsah_reset:
+                    try:
+                        delattr(args, reset_arg)
+                    except AttributeError:
+                        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(AttributeError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(AttributeError)`

(SIM105)